src/COMDIRAC/Interfaces/scripts/dchmod.py

```diff
@@ -81,7 +79,7 @@ def main():
 
     for lfn in lfns:
         try:
-            pathDict = {lfn: eval("0" + mode)}
+            pathDict = {lfn: eval("0o" + mode)}
```

Comment:
Rather than using eval:
```suggestion
            pathDict = {lfn: int(mode, base=8)}
```